README.md

```diff
@@ -9,21 +9,27 @@ https://github.com/mongodb/docs-java
 
 ## Build Instructions
 
+:warning: **When building docs for a new version, make sure to update the following:**
+- `<this repo>/reference/config.toml` to point to the new base version
+- `<this repo>/landing/data/releases.toml` to link to the docs for the new version
+- any redirects such as the [Java sync docs redirect](https://github.com/mongodb/docs-java-other/pull/3/files#diff-0f1a8692163867f83ff7451f3018bae71d3d16dbee396abf03263784e5dda940)
+
 After making required changes, submit a pull request. Once your PR is approved and merged, run the `publish-docs` script with the version:
 
 ```sh
 ./publish-docs <version>
 ```
 
 This will initiate a submodule (if necessary) that tracks the *gh-pages* branch
-of the [mongo-java-driver](https://github.com/mongodb/mongo-java-driver). It will
-then build the documents and copy items over as necessary (using rsync).
+of the [mongo-java-driver](https://github.com/mongodb/mongo-java-driver). It then
+builds the documents in a new directory in the submodule that corresponds to the
+new version name.
 
 ## Building API Documentation
 
 For major and minor releases, it is required to build the api docs. This
 can be done by navigating to your `mongo-java-driver` repo (*Note: NOT the submodule in this repo*) and executing the appropriate `gradlew` command
-after checking out the correct tag.
+after checking out the correct tag. Make sure you installed the Java version specified in the `:bson:compileJava` task on your machine.
```

Comment:
Issue: [Machine](https://docs.mongodb.com/meta/style-guide/terminology/alphabetical-terms/#term-machine) should only be used to refer to a virtual machine

```suggestion
after checking out the correct tag. Make sure you installed the Java version specified in the `:bson:compileJava` task on your computer.
```